documentation/modules/exploit/unix/webapp/trixbox_ce_auth_rce.md

```diff
@@ -13,7 +13,7 @@ Used the Trixbox CE v2.8.0.4 ISO which is available for downloading on sourcefor
 ```
 Id  Name
 --  ----
-0   Trixbox CE <= v2.8.0.4
+0   Trixbox CE v2.8.0.4
```

Comment:
If this affects older versions and you have confirmed this, please revert this change. Only the title of the exploit module needed to be changed.

Reply:
Confirmed reverted.